src/gui/userstatusselectormodel.cpp

```diff
@@ -406,23 +406,23 @@ QString UserStatusSelectorModel::timeDifferenceToString(int differenceSecs) cons
     } else if (differenceSecs < 60 * 60) {
         const auto minutesLeft = std::ceil(differenceSecs / 60.0);
         if (minutesLeft == 1) {
-            return tr("1 minute");
+            return tr("%1 minute", "", minutesLeft).arg(minutesLeft);
```

Comment:
this is not making sense because you use a different string here
translators will easily get confused
why not keeping it like it was before with `1 minute` ?
as a translator, I would expect a typo when seeing `%1 minute` and `%1 minutes`

Reply:
I mean seeing both those string top translate, I would expect a typo in the first one and would always translate it as a plural form

Reply:
I mean that as a translator I would always translate with a plural form expecting a typo in the first string

Reply:
@mgallien  Would instead:

```
        if (minutesLeft == 1) {
            return tr("%1 minute", "", minutesLeft).arg(minutesLeft);
        } else {
            return tr("%1 minutes", "", minutesLeft).arg(minutesLeft);
        }
```
Replace it with:

`return tr("%1 minute", "%1 minutes", minutesLeft).arg(minutesLeft);`

As I understand correctly, why give the condition == 1. The text string will match itself singular or plural. Unless I'm wrong.

Reply:
no that will break translations when `minutesLeft` is equal to `1`
keep it like it was before and just modify it for plurals form

Reply:
> Replace it with:
> 
> `return tr("%1 minute", "%1 minutes", minutesLeft).arg(minutesLeft);`
the third parameter is not meant for plurals form
that change is wrong